zoom_monitor.py

```diff
@@ -150,92 +172,74 @@ def _open_participants_list(self):
-                    outer.find_element(By.XPATH, PARTICIPANTS_BTN)
-                except NoSuchElementException:
-                    self._logger.debug("participants not present, next...")
-                    continue  # wrong footer element, try the next one
-
-                try:
-                    # For reasons we haven't figured out yet, something
-                    # changed in late 2023 so that clicking on the participants
-                    # list merely causes the button to be selected, not fully
-                    # clicked. As a small clue: if a human clicks on it, the
-                    # mouse-down makes the button selected, and the mouse-up
-                    # actually opens the participants list. We haven't tracked
-                    # down exactly what's going wrong, but double-clicking on
-                    # it seems to work okay (and Alan suspects that the second
-                    # click implicitly creates a mouse-up on the first one,
-                    # and that's the important part).
-                    # If the button is already selected, only one click is
-                    # needed.
-                    outer.click()
-                    if not hovering:
-                        outer.click()  # Channeling our inner grandma
-                    self._logger.debug("participants list clicked")
-                except ElementClickInterceptedException:
-                    self._logger.debug("DOM isn't set up; wait and try again")
-                    time.sleep(1)  # The DOM isn't set up; wait a little longer
-                    break  # Go to the next overall attempt
-
-                # Now that we've clicked the participants list without raising
-                # an exception, wait until it shows up.
-                self._wait_for_element(
-                    By.CLASS_NAME, "participants-wrapper__inner")
-                self._logger.info("participants list opened")
-                return  # Success!
+            button = self._find_participants_button()
+            try:
+                # Clicking on the participants list only selects the button.
+                # As a small clue: if a human clicks on it, the mouse-down
+                # selects the button while the mouse-up opens the participants
+                # list. Double-clicking on it seems to work okay (maybe the
+                # second click implicitly creates a mouse-up on the first one).
+                # If the button is already selected, only one click is needed.
+                button.click()
+                if not selected:
+                    button.click()  # Channeling our inner grandma
+                self._logger.debug("participants list clicked")
+            except ElementClickInterceptedException:
+                self._logger.debug("DOM isn't set up; wait and try again")
+                time.sleep(1)  # The DOM isn't set up; wait a little longer
```

Comment:
While we're cleaning up, remove this comment, which is redundant with the previous line! :sweat_smile: